lib/src/pages/yaru_tabbed_page.dart

```diff
@@ -21,8 +21,8 @@ class YaruTabbedPage extends StatefulWidget {
     this.onTap,
   });
 
-  /// A list of [IconData] used inside the tabs - must have the same length as [tabTitles] and [views].
-  final List<IconData> tabIcons;
+  /// A list of [Icon]s used inside the tabs - must have the same length as [tabTitles] and [views].
+  final List<Icon> tabIcons;
```

Comment:
Here I think the type should be List<Widget>
Example: animated icons are widgets.

Reply:
Ah yes, that makes a lot of sense - thanks!